src/galaxy/SystemPath.cpp

```diff
@@ -42,11 +42,13 @@ SystemPath SystemPath::Parse(const char *const str)
 
 	assert(s.length() > 0);
 
+	// delete Leading '('
 	if (s[0] == '(')
 		s = s.substr(1, s.length());
```

Comment:
Oh yeah, that was just outside the scope here since there were hundreds of things to go through. There's a lot of errors and warnings too but sadly there's a lot of noise so I just tackled this low hanging stuff first.